src/expression/visibility.ts

```diff
@@ -0,0 +1,43 @@
+import {createExpression, findGlobalStateRefs} from '.';
+import {type GlobalProperties, type StylePropertySpecification} from '..';
+import {type VisibilitySpecification} from '../types.g';
+
+const visibilitySpec: StylePropertySpecification = {
+    type: 'enum',
+    'property-type': 'data-constant',
+    expression: {
+        interpolated: false,
+        parameters: ['global-state']
+    },
+    values: {visible: {}, none: {}},
+    transition: false,
+    default: 'visible'
+};
+
+export default function createVisibility(visibility: VisibilitySpecification, globalState: Record<string, any>) {
```

Comment:
What is the return type of this method? What is it doing? Where are the tsdoc comments on it?